docs/community_resources/release_process.md

```diff
@@ -19,18 +25,19 @@ We usually release [[= product_name =]] four times a year following the seasons
 
 ## Versioning conventions
 
-Both [[= product_name =]] editions use [semantic versioning](http://semver.org/).  
+All [[= product_name =]] editions use [semantic versioning](http://semver.org/).  
 
 The version number of [[= product_name =]] and all its internal components follows the semantic versioning conventions: vX.Y.Z.
 
-- Changes to X indicate breaking changes. They usually concern mostly internal things, but developers should check in our change logs if they need to adjust their code to continue using the API or features. If there are larger breaks (like the new Back Office in v2), this is announced well in advance of the upcoming release.
+- Changes to X indicate breaking changes. They usually concern mostly internal things, but developers should check in our change logs if they need to adjust their code to continue using the API or features. If there are larger breaks, this is announced well in advance of the upcoming release.
 - Y represents new features and functionalities.
 - Z represents patches, bug fixes, smaller improvements, etc.
 
-Distribution files of our two editions are as follows:
+Distribution files of our three editions are as follows:
 
-- for [[= product_name =]]: ezplatform-vX.Y.Z.tgz
+- for [[= product_name_content =]]: ezplatform-vX.Y.Z.tgz
```

Comment:
We will change the name of the tarball as well. I guess it will be `ibexa-content-vX.Y.Z.tgz`

Reply:
```suggestion
- for [[= product_name_content =]]: ibexa-content-vX.Y.Z.tgz
```